test/atomicrmw_fadd.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_EXT_shader_atomic_float_add %t.bc -o %t.spv
```

Comment:
@sarnex 
Sorry, I have to withdraw my approve. Please add checks in the test like:
CHECK-DAG: Extension "SPV_EXT_shader_atomic_float_add"
CHECK-DAG: Capability AtomicFloat32AddEXT

I believe, that this checks will fail, since we probably need to add them explicitly. Also there should be a separate test for double to emit AtomicFloat64AddEXT capability.

Please also move this (or these two tests) into test/extensions/EXT/SPV_EXT_shader_atomic_float_/ directory

Reply:
@MrSidims Capability stuff seems to work. I believe I addressed all of this in my latest commit